x-pack/platform/plugins/shared/streams/server/routes/internal/streams/crud/route.ts

```diff
@@ -84,7 +84,7 @@ export const streamDetailRoute = createServerRoute({
     const streamEntity = await streamsClient.getStream(params.path.name);
 
     const indexPattern = Streams.GroupStream.Definition.is(streamEntity)
-      ? streamEntity.group.members.join(',')
+      ? streamEntity.group.relationships.map((relationship) => relationship.name).join(',')
```

Comment:
If we have a relationship to another group stream, its index pattern is not useful here. we should either exclude it or include the group members as well. In both cases we need to fetch the members to find their type, with the second option needing a graph traversal

Should we instead not allow group <-> group relationships for now ?

Reply:
I think I just changed this for type reasons, I don't think this API is used by Group Streams so I'll just return 0 here since the doc count for a Group Stream doesn't make so much sense.

If we wanted to do this, we would indeed need ES|QL views to manage the layers.